Fix search order in man.conf so that system installed manpages
will be found first ...

--- man-1.5p/src/man.conf.in
+++ man-1.5p/src/man.conf.in
@@ -36,11 +36,11 @@
 #
 # Every automatically generated MANPATH includes these fields
 #
-MANPATH	/usr/man
 MANPATH	/usr/share/man
-MANPATH	/usr/local/man
 MANPATH	/usr/local/share/man
 MANPATH	/usr/X11R6/man
+MANPATH	/usr/local/man
+MANPATH	/usr/man
 #
 # Uncomment if you want to include one of these by default
 #
